test/testobjects/TestOrchestrations.ts

```diff
@@ -165,7 +161,7 @@ export class TestOrchestrations {
     public static SayHelloWithActivityYieldTwice: any = df.orchestrator(function*(context: any) {
```

Comment:
I think we can replace the `any` type with `IOrchestrationFunctionContext` like in `SendHttpRequest`, but we can punt that to another PR.

Reply:
I'd look at that in the other PR I'm working on for improving the typing.